### ISSUE TITLE
ParameterName "timeout" does not work with elasticache_cache_parameter_group

### ISSUE THREAD

**Issue by htnosm** [k1LoW/awspec#351]
Specifying timeout parameter on elasticache_cache_parameter_group fails.

```
its': Object#timeout is deprecated, use Timeout.timeout instead.
```

It seems to be recognized as ruby or rspec's timeout.
Is there a way to use parameter name 'timeout'?

e.g) "repl-timeout" works but "timeout" fails
```
describe elasticache_cache_parameter_group('default.redis3.2') do
  its(:"repl-timeout") { should eq "60" }
  its('repl-timeout') { should eq "60" }
  its(:"repl-timeout") { should eq "0" }
  its(:"timeout") { should eq "0" }
  its('timeout') { should eq "0" }
end
```

```
Failures:

  1) elasticache_cache_parameter_group 'default.redis3.2' repl-timeout should eq "0"
     Failure/Error: its(:"repl-timeout") { should eq "0" }

       expected: "0"
            got: "60"

       (compared using ==)
     # ./spec/elasticache_cache_parameter_group_test_spec.rb:8:in `block (2 levels) in <top (required)>'

  2) elasticache_cache_parameter_group 'default.redis3.2' timeout
     Failure/Error: its(:"timeout") { should eq "0" }

     ArgumentError:
       wrong number of arguments (given 0, expected 1..2)
     # ./spec/elasticache_cache_parameter_group_test_spec.rb:9:in `block (2 levels) in <top (required)>'

  3) elasticache_cache_parameter_group 'default.redis3.2' timeout
     Failure/Error: its('timeout') { should eq "0" }

     ArgumentError:
       wrong number of arguments (given 0, expected 1..2)
     # ./spec/elasticache_cache_parameter_group_test_spec.rb:10:in `block (2 levels) in <top (required)>'
```

**Comment by k1LoW** [comment]
Hi @htnosm Thank you for your report !!

**Comment by k1LoW** [comment]
Released as v1.4.2.
